doc/dev/cephx.rst

```diff
@@ -0,0 +1,353 @@
+=====
```

Comment:
if you add an anchor here, it would be useful when other sections of the docs want to use the `:ref:` 
```suggestion
.. _cephx:

=====
```